index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/b42bc5ce06.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!-- navigation -->
+    <header class="header">
+        <nav id="nav">
+            <div class="nav-center">
+                <a href="index.html">
+                    <div id="JohnDoe">
+                        <h3>John Doe</h3>
+                    </div>
+                </a>
+                <div id="bars">
+                    <i class="fas fa-bars"></i> 
+                </div>
+            </div>
+        </nav>
+</header>
+<!-- end of navigation -->
+<!-- about section -->
+<section class="mainbody">
+<div class="title">
+    <h1>Hey There.
+        I'm Ghulam Shabbir</h1>

```

Comment:
- [ ] Great job so far :muscle: However, I noticed that your design has a bigger font size for the title compared to the Figma design. Also, per the Figma design, the title only extends to two lines and not 3 as you currently have in your design :wink: 
- [ ] Please ensure that your design follows the design guidelines specified in the Figma design so your design looks like the original design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/226998837-de6e1aa6-072c-4b10-b22f-2d066133d207.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/226999517-2efae313-92a6-4901-893c-c39ce17303da.png)